channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/ProcessCompatible.kt

```diff
@@ -4,29 +4,29 @@ import com.justai.jaicf.channel.http.asHttpBotRequest
 import com.justai.jaicf.channel.jaicp.dto.JaicpBotRequest
 import com.justai.jaicf.channel.jaicp.dto.JaicpBotResponse
 import com.justai.jaicf.channel.jaicp.dto.fromRequest
-import kotlinx.serialization.json.JsonElement
-import kotlinx.serialization.json.json
-import kotlinx.serialization.json.jsonArray
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.*
 
 internal fun JaicpCompatibleBotChannel.processCompatible(
     botRequest: JaicpBotRequest
 ): JaicpBotResponse {
     val startTime = System.currentTimeMillis()
     val request = botRequest.raw.asHttpBotRequest(botRequest.stringify())
     val response = process(request)?.let { response ->
-        val rawJson = JSON.parseJson(response.output.toString())
+        val rawJson = JSON.decodeFromString<JsonObject>(response.output.toString())
         addRawReply(rawJson)
     } ?: throw RuntimeException("Failed to process compatible channel request")
 
     val processingTime = System.currentTimeMillis() - startTime
     return JaicpBotResponse.fromRequest(botRequest, response, processingTime)
 }
 
-private fun addRawReply(rawResponse: JsonElement) = json {
-    "replies" to jsonArray {
-        +(json {
-            "type" to "raw"
-            "body" to rawResponse
+private fun addRawReply(rawResponse: JsonElement) = buildJsonObject {
+    put("replies", buildJsonArray {
```

Comment:
There are ready to use `putJsonArray` and `putJsonObject` methods, that do exactly the same